src/app/backend-api.service.ts

```diff
@@ -2125,6 +2136,17 @@ export class BackendApiService {
     });
   }
 
+  UploadVideo(endpoint: string, PublicKeyBase58Check: string, FileSize: number): Observable<any> {
```

Comment:
I don't actually see where this is used.  It looks like the route path is used by tus.Upload() but this function isn't actually used.  Is it needed?

Reply:
yeah we just use the routepath - I'll delete this.